packages/vaadin-date-picker/test/validation.test.js

```diff
@@ -0,0 +1,164 @@
+import { expect } from '@esm-bundle/chai';
```

Comment:
This test suite seems to overlap with `form-input.test.js` - please remove one of these.

Reply:
Done